packages/quantic/force-app/main/default/lwc/quanticResultQuickview/e2e/fixture.ts

```diff
@@ -0,0 +1,90 @@
+import {ResultQuickviewObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+  searchHtmlRequestRegex,
+  insightQuickviewRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+import {SearchObjectWithQuickview} from '../../../../../../playwright/page-object/searchObjectWithQuickview';
+
+const resultQuickviewUrl = 's/quantic-result-quickview';
+
+interface ResultQuickviewOptions {
+  result: string;
+  maximumPreviewSize: number;
+  previewButtonIcon: string;
+  previewButtonLabel: string;
+  previewButtonVariant: string;
+  tooltip: string;
+}
+
+type QuanticResultQuickviewE2EFixtures = {
+  resultQuickview: ResultQuickviewObject;
+  search: SearchObjectWithQuickview;
+  options: Partial<ResultQuickviewOptions>;
+};
+
+type QuanticResultQuickviewE2ESearchFixtures =
+  QuanticResultQuickviewE2EFixtures & {
+    urlHash: string;
+  };
+
+type QuanticResultQuickviewE2EInsightFixtures =
+  QuanticResultQuickviewE2ESearchFixtures & {
+    insightSetup: InsightSetupObject;
+  };
+
+export const testSearch =
+  quanticBase.extend<QuanticResultQuickviewE2ESearchFixtures>({
+    options: {},
+    urlHash: '',
+    search: async ({page}, use) => {
+      await use(
+        new SearchObjectWithQuickview(
+          page,
+          searchRequestRegex,
+          searchHtmlRequestRegex
+        )
+      );
+    },
+    resultQuickview: async ({page, options, configuration, urlHash}, use) => {
+      await page.goto(
+        urlHash ? `${resultQuickviewUrl}#${urlHash}` : resultQuickviewUrl
```

Comment:
I dont think you need urlHash here as I dont see any test that require checking or passing something in the url